### ISSUE TITLE
Exception thrown on using ordered() scope

### ISSUE THREAD

**Issue by Propaganistas** [spatie/eloquent-sortable#5]
`Argument 1 passed to App\Model::scopeOrdered() must be an instance of Illuminate\Database\Query\Builder, instance of Illuminate\Database\Eloquent\Builder given`

I have defined scopes myself in the model and seems I didn't use any typehinting at all for `$query`. So this could be a possible fix?

**Comment by freekmurze** [comment]
Fixed in version 1.1.1 by removing the typehint. I will add the typehint later when I'm sure which one to use in L4 and L5.

**Comment by Propaganistas** [comment]
Don't forget the interface.

`Declaration of Spatie\EloquentSortable\Sortable::scopeOrdered($query) must be compatible with Spatie\EloquentSortable\SortableInterface::scopeOrdered(Illuminate\Database\Query\Builder $query)`

**Comment by freekmurze** [comment]
Fixed in 1.1.2. Sorry about that.

**Comment by uyab** [comment]
Hi @freekmurze, look like you are adding the typehint again, and the same error occured.

```
Argument 1 passed to App\Model\Post::scopeOrdered() must be an instance of Illuminate\Database\Query\Builder, instance of Illuminate\Database\Eloquent\Builder given 
```

**Comment by freekmurze** [comment]
Thanks for reporting this. Fixed it again in 2.0.1 and I've also added an automated test so you won't see this bug again.
